.github/test-spec.yml

```diff
@@ -269,21 +269,19 @@
   - "boards/nordic/*kbd/**/*"
   - "boards/nordic/*dongle/**/*"
   - "cmake/*"
-  - "drivers/sensor/pmw3360/**/*"
   - "drivers/sensor/paw3212/**/*"
+  - "drivers/sensor/pmw3360/**/*"
   - "dts/bindings/sensor/pixart*"
-  - "include/bluetooth/services/**/*"
-  - "include/bluetooth/*"
+  - "include/bluetooth/**/*"
   - "include/caf/**/*"
   - "include/sensor/*"
-  - "include/app_event_manager.h"
-  - "include/nrf_profiler.h"
   - "scripts/hid_configurator/**/*"
-  - "subsys/bluetooth/services/**/*"
-  - "subsys/bluetooth/adv_prov/**/*"
+  - "subsys/app_event_manager/**/*"
   - "subsys/bluetooth/*"
+  - "subsys/bluetooth/adv_prov/**/*"
```

Comment:
what about the bootloader, MPSL, SUIT, and SDFW services in the subsys category?

Reply:
This PR is only meant to introduce basic fixes. We have a separate task for further improvements: https://nordicsemi.atlassian.net/browse/NCSDK-23274 (we need changes in both `sdk-nrf` and `sdk-zephyr`).
Maybe I could pull this ticket into ongoing sprint to speed up introducing further CI improvements?

Reply:
Thanks, we will properly fix the configuration as part of the linked ticket. This PR covers the basics and it's good to merge it to minimize the number of the nRF Desktop CI failures.